src/primitives.rs

```diff
@@ -18,6 +18,7 @@ pub type Domain = Vector<u8, 32>;
 pub const GENESIS_SLOT: Slot = 0;
 pub const GENESIS_EPOCH: Epoch = 0;
 pub const FAR_FUTURE_EPOCH: Epoch = Epoch::MAX;
+pub const BASE_REWARDS_PER_EPOCH: usize = 4;
```

Comment:
this is _actually_ a phase 0 constant, so it is better imo to put it there...

could go in `src/phase0/mod.rs` even

Reply:
fixed